CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 - Remove `options.autoAppStart` and `setAppStartEnd` ([#2680](https://github.com/getsentry/sentry-dart/pull/2680))
 - Add hint for transactions ([#2675](https://github.com/getsentry/sentry-dart/pull/2675))
   - `BeforeSendTransactionCallback` now has a `Hint` parameter
+- Remove max response body size ([#2709](https://github.com/getsentry/sentry-dart/pull/2709))
+  - Responses are now only attached if size is below ~15mb
```

Comment:
and also pls mention that this is currently only applicable for the dio integration, not our http integration

Reply:
our http integration doesn't support adding the response data generally, we could add that feature later at some point